scintilla/win32/Scintilla.vcxproj

```diff
@@ -95,6 +94,7 @@
     </ClCompile>
     <Link>
       <LinkTimeCodeGeneration>Default</LinkTimeCodeGeneration>
+      <CETCompat>true</CETCompat>
```

Comment:
Why is this line added?

Reply:
See Release Notes (https://www.scintilla.org/ScintillaHistory.html):
- Fix ARM64 builds with Visual C++ due to unsupported CETCOMPAT flag. [Bug #2324](https://sourceforge.net/p/scintilla/bugs/2324/).

The setting is already there at the common section, so it was just splitted out to the individual settings for x86, x64 because the feature is not available for arm64 (see also https://docs.microsoft.com/en-us/cpp/build/reference/cetcompat?view=msvc-170)